Classifiers/classifier-Prisma_Cloud.json

```diff
@@ -5,12 +5,518 @@
 	"fromVersion": "5.0.0",
 	"id": "230ea712-b969-4d73-8e1e-a1360ab8942c",
 	"keyTypeMap": {
-		"Prisma Cloud": "Prisma Cloud"
+                "168bfaa0-8c1d-427e-bfa8-4d96d82e3d83": "AWS IAM Policy Misconfiguration",
+                "31626ca9-f659-4d25-9d88-fa32262bbba7": "AWS IAM Policy Misconfiguration",
+                "9a5813af-17a3-4058-be13-588ea00b4bfa": "AWS IAM Policy Misconfiguration",
+                "Prisma Cloud": "Prisma Cloud",
+                "a2107824-6ed5-4c67-9450-8b154bb1fd2b": "AWS IAM Policy Misconfiguration",
+                "a8dcc272-0b02-4534-8627-cf70ddd264c5": "AWS IAM Policy Misconfiguration",
+                "b1acdeff-4959-4c14-8a5e-2adc1016a3d5": "AWS IAM Policy Misconfiguration",
+                "ef7c537b-72eb-42a7-bab7-cb2d22c76a0d": "AWS IAM Policy Misconfiguration",
+                "f53107a2-00b2-46fb-98a9-1f12262c7d44": "AWS IAM Policy Misconfiguration",
+                "fd4dae57-509e-4374-96d3-e136821fc3f3": "AWS IAM Policy Misconfiguration"
 	},
 	"mapping": {
+                "AWS IAM Policy Misconfiguration": {
+                        "dontMapEventToLabels": false,
+                        "internalMapping": {
+                                "Account": {
```

Comment:
Same as for the account field here. If you install new content you'll see the classifier has changed. please change it (to account name) according to the fixed classifier.